app/views/supplies/index.html.haml

```diff
@@ -1,12 +1,26 @@
-%table.table.table--bordered.table--striped.table--linked
+= link_to "Add New Supply", new_supply_path, class: "btn btn-default"
+
+%table.table.table-striped.table-linked.table-condensed
   %thead
     %tr
       %th Name
       %th Short Code
+      %th
   %tbody
     - @supplies.each do |supply|
-      %tr
+      %tr{ class: supply.orderable? ? "" : "danger" }
         %td
-          %span.cell--primary= supply.name
+          -if supply.orderable
```

Comment:
on line `21` there is a question mark/inflection on `supply.orderable?` Did you mean to put that here, as well?